src/Arcus.Scripting.IntegrationAccount/Scripts/Set-AzIntegrationAccountPartners.ps1

```diff
@@ -0,0 +1,91 @@
+Param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Resource group name is required"),
+    [Parameter(Mandatory = $true)][string] $Name = $(throw "Name of the Integration Account is required"),
+    [parameter(Mandatory = $false)][string] $PartnerFilePath = $(if ($PartnersFolder -eq '') { throw "Either the file path of a specific partner or the file path of a folder containing multiple partners is required, e.g.: -PartnerFilePath 'C:\Partners\partner.json' or -PartnersFolder 'C:\Partners'" }),
+    [parameter(Mandatory = $false)][string] $PartnersFolder = $(if ($PartnerFilePath -eq '') { throw "Either the file path of a specific partner or the file path of a folder containing multiple partners is required, e.g.: -PartnerFilePath 'C:\Partners\partner.json' or -PartnersFolder 'C:\Partners'" }),
+    [Parameter(Mandatory = $false)][string] $ArtifactsPrefix = ''
+)
+
+if ($PartnerFilePath -ne '' -and $PartnersFolder -ne '') {
+    throw "Either the file path of a specific partner or the file path of a folder containing multiple partners is required, e.g.: -PartnerFilePath 'C:\Partners\partner.json' or -PartnersFolder 'C:\Partners'"
+}
+
+function UploadPartner {
+    param
+    (
+        [Parameter(Mandatory = $true)][System.IO.FileInfo] $Partner
+    )
+
+    $partnerData = Get-Content -Raw -Path $Partner.FullName | ConvertFrom-Json
+
+    $partnerName = $partnerData.name
+    if ($partnerName -eq $null -or $partnerName -eq '') {
+        throw 'Partner name is empty'
+    }
+
+    if ($ArtifactsPrefix -ne '') {
+        $partnerName = $ArtifactsPrefix + $partnerName
+    }
+    Write-Host "Uploading partner '$partnerName' into the Integration Account '$Name'"
+
+    $businessIdentities = $null
+    ForEach ($businessIdentity in $partnerData.properties.content.b2b.businessIdentities) {
+        $qualifier = $businessIdentity.qualifier
+        $value = $businessIdentity.value
+
+        $businessIdentities += ,@("$qualifier","$value")
```

Comment:
Oh, this is new syntax for me. What does the heading comma mean here?

Reply:
That was new to me too, the comma makes sure the value is correctly added as an array item

Reply:
So, it would otherwise be... appended to the array, instead of a separate item?

Reply:
It is to add an array with 2 elements, trying to find some official docs on this but not finding much yet, it is mentioned here: https://devblogs.microsoft.com/powershell/array-literals-in-powershell/: `$a += ,(5,6)      # add another element which is an array of two element. `